src/clusterfuzz/_internal/bot/tasks/utasks/fuzz_task.py

```diff
@@ -548,7 +550,8 @@ def sync_from_gcs(self):
 
     # Check if the corpus was recently synced. If yes, set a flag so that we
     # don't sync it again and save some time.
-    if last_sync_time and os.path.exists(self._corpus_directory):
+    if not environment.is_uworker() and last_sync_time and os.path.exists(
```

Comment:
add a TODO to completely remove this code once we are completely migrated?

Reply:
Done.